contracts/RenPool.sol

```diff
@@ -1,35 +1,43 @@
 pragma solidity ^0.8.0;
 
-import "OpenZeppelin/openzeppelin-contracts@4.0.0/contracts/token/ERC20/ERC20.sol";
+import "./RenToken.sol";
+
```

Comment:
still same issue. You are adding 791 lines and removing 39. Too many changes. Look at the files tab in your PR.
I'd say let's keep this PR for later and focus on the other stuf